modules/developer_manual/pages/_partials/webdav_api/search/file_properties.adoc

```diff
@@ -25,19 +25,20 @@
 |`DAV`
 
 |lockdiscovery
-|
+| Supports the "_persistent file locking_" backend feature. 
+For more information on this property, please refer to https://tools.ietf.org/html/rfc4918#section-6.8[the W3C spec].
 |`DAV`
 
 |resourcetype
-|The type of the resource, e.g., collection.
+|The resource's type. If the resource is a folder, then the value is set to `collection`, otherwise, no value is returned.
```

Comment:
Note FYI: for some app related Webdav implementations like custom groups, we have some custom resource types in place there. I don't think it's necessary to document those here.